src/sandip/Test_1/Program2.java

```diff
@@ -0,0 +1,27 @@
+package sandip.Test_1;
+
+public class Program2 {
+	int count;
+	char store;
+
+	void nonRepeatingChar(String input) {
+		for (int index = 0; index < input.length() - 1; index++) {
+			count = 1;
+			store = input.charAt(index);
+			for (int index1 = index + 1; index1 < input.length() - 1; index1++) {
+				if (store == input.charAt(index1)) {
+					count++;
+				}
+			}
+			if (count == 1) {
```

Comment:
here you can aslo use 
indexOf(ch) ==lastIndexOf(ch) logic to avoid one more for loop